integration-tests/validator/src/main/java/org/apache/camel/quarkus/component/validator/it/ValidatorRouteBuilder.java

```diff
@@ -21,7 +21,14 @@
 public class ValidatorRouteBuilder extends RouteBuilder {
     @Override
     public void configure() {
-        from("direct:start")
+        from("direct:classpath")
                 .to("validator:message.xsd");
+
+        from("direct:filesystem")
+                .to("validator:file:src/main/resources/message.xsd");
+
+        from("direct:http")
+                .toD("validator:https://raw.githubusercontent.com/apache/camel-quarkus/main/integration-tests/validator/src/main/resources/message.xsd");
```

Comment:
This might be risky for several reasons: 

* If we change message.xsd in the future, the tests in older branches may start failing
* GitHub might be inaccessible in some environments
* GitHub's availability will impact the reliability of this test

I'd vote for serving message.xsd from an ad hoc WireMock server. There are plenty of examples in the source tree.